docs/api/structures/user-agent-metadata.md

```diff
@@ -0,0 +1,12 @@
+# UserAgentMetadata Object
```

Comment:
Can we talk a bit more about the UA Client Hints? https://wicg.github.io/ua-client-hints/#ref-for-dictdef-uadatavalues

Reply:
Just added both the MDN and W3C links.